README.md

```diff
@@ -132,8 +133,10 @@ Note that you may wish to customize your schema results more than this module pr
 
 Flask configuration:
 
-* `FLASK_PYDANTIC_API_RENDER_ERRORS`: bool = True.  If true, pydantic validation errors will be rendered to json and returned as a normal response.  If false, pydantic errors will yield a standard ValidationError exception.
+* `FLASK_PYDANTIC_API_RENDER_ERRORS`: bool = False.  If true, pydantic validation errors will be rendered to json and returned as a normal response.  If false, pydantic errors will yield a standard ValidationError exception.
```

Comment:
Why change this to false?  I do not see a corresponding change in the code, but why change the default anyway?